skimage/feature/corner.py

```diff
@@ -915,16 +916,35 @@ def corner_subpix(image, corners, window_size=11, alpha=0.99):
 
 def corner_peaks(image, min_distance=1, threshold_abs=None, threshold_rel=None,
                  exclude_border=True, indices=True, num_peaks=np.inf,
-                 footprint=None, labels=None, *, num_peaks_per_label=np.inf):
+                 footprint=None, labels=None, *, num_peaks_per_label=np.inf,
+                 p=np.inf):
     """Find corners in corner measure response image.
 
     This differs from `skimage.feature.peak_local_max` in that it suppresses
     multiple connected peaks with the same accumulator value.
 
     Parameters
     ----------
+    image : ndarray
+        Input image.
+    min_distance : int, optional
+        The minimum distance seperating peaks. Use the ``p`` argument
+        to set the Minkowski p-norm defining the distance.
     * : *
         See :py:meth:`skimage.feature.peak_local_max`.
+    p : float
+        Which Minkowski p-norm to use. Should be in the range [1, inf].
+        A finite large p may cause a ValueError if overflow can occur.
+        inf corresponds to the Chebyshev distance and 2 to the
+        euclidean distance.
+
+    Returns
+    -------
+    output : ndarray or ndarray of bools
+
+        * If `indices = True`  : (row, column, ...) coordinates of peaks.
+        * If `indices = False` : Boolean array shaped like `image`, with peaks
```

Comment:
I think these `* If (...)` would be better in a `Notes` section... is it just me?

Reply:
I personally have no strong opinion on this, but I don't know what to put in `output` description then...

Reply:
Btw, we are in the 3. of https://github.com/scikit-image/scikit-image/issues/3892#issuecomment-591819705 case here ;-)

Reply:
@alexdesiqueira, I see here that I simply copied the Return section of [`peak_local_max`](https://scikit-image.org/docs/dev/api/skimage.feature.html#skimage.feature.peak_local_max).

Reply:
I'd probably leave it here, since it describes the return; in the notes section, we typically discuss implementation details that users may be curious about, or that would help them better understand the algorithm.